src/DOM.js

```diff
@@ -0,0 +1,268 @@
+    toDosView.append(addToDoButtonContainer);
+
+    project.toDos.forEach(toDo => {
+      addToDo(toDo, toDoClickHandler);
+    });
+
+    if (project.toDos.length === 0) {
+      singleToDoView.innerHTML = '';
+      return null;
+    } else {
+      let toDoElement = toDosList.children[0];
+      toDoElement.classList.add('active');
+      setActiveToDo(project.toDos[0], singleToDoClickHandlers)
+    }
+  }
+
+  function addToDo(toDo, toDoClickHandler) {
+    const a = document.createElement('a');
+    const priority = document.createElement('span');
+    const title = document.createElement('span');
+    const date = document.createElement('span');
+    const priorityBadgeClass = getPriorityBadgeClass(toDo.priority);
+
+    priority.classList.add('badge', `badge-${ priorityBadgeClass }`);
+    priority.innerHTML = '&nbsp;';
+    priority.title = `Priority: ${ toDo.priority }`;
+
+    title.classList.add('title', 'flex-fill', 'mx-2');
+    title.innerHTML = toDo.title;
+
+    date.innerHTML = toDo.dueDate ? `(due on ${ format(toDo.dueDate, 'DD/MM/YYYY') })` : '';
+    date.classList.add('small', 'due-date');
+    if (isPast(toDo.dueDate)) {
+      date.classList.add('text-danger', 'font-weight-bold');
+    }
+
+    a.href = '#';
+    a.dataset.toDoId = toDo.id;
+    a.append(priority);
+    a.append(title);
+    a.append(date);
+    a.classList.add('list-group-item',
+                    'list-group-item-action',
+                    'd-flex',
+                    'justify-content-between',
+                    'align-items-center');
+    if (toDo.completed) {
+      a.classList.add('completed');
+    }
+    a.addEventListener('click', toDoClickHandler)
+    toDosList.append(a);
+  }
+
+  function setActiveToDo(toDoObject, singleToDoClickHandlers) {
+    const toDoElement = toDosList.querySelector(`[data-to-do-id="${toDoObject.id}"]`);
+    const currentActive = toDoElement.parentElement.querySelector('.active');
+    if (currentActive) currentActive.classList.remove('active');
+    toDoElement.classList.add('active');
+
+    renderSingleToDo(toDoObject, singleToDoClickHandlers);
+  }
+
+  function getPriorityBadgeClass(priority) {
+    let className = '';
+    switch (priority) {
+      case 'regular':
+        className = 'success';
+        break;
+      case 'high':
+        className = 'danger';
+        break;
+      case 'low':
+        className = 'secondary';      
+    }
+    return className;
+  }
+
+  function renderSingleToDo(toDo, singleToDoClickHandlers) {
+    const completeButton = document.createElement('button');
+    const completeButtonContainer = document.createElement('div');
+    const form = document.createElement('form');
+    const deleteButton = document.createElement('button');
+    const updateButton = document.createElement('button');
+    const formButtonsContainer = document.createElement('div');
+    const dateValidationRegEx = '^\\d{2}/\\d{2}/\\d{4}$';
+
+    const priority = {
```

Comment:
Why not `===` ?

Reply:
Not sure where exactly you are pointing... Anyway, I'd like to ask - why yes?

Reply:
[Here](https://github.com/nikitin2009/todo-list/blob/2577ff117a9acaf792bed8cb586f9fd867e9e83a/src/DOM.js#L168-L172), I saw 
```javascript
    const priority = {
      low: toDo.priority == 'low',
      regular: toDo.priority == 'regular',
      high: toDo.priority == 'high'
    }
```
Just want to understand why we can't use `===` ? Again, this is just according to javascript style guide.

Reply:
OK, here I can use it.